setup.py

```diff
@@ -38,6 +38,7 @@
     "numpy~=1.20",
     "matplotlib~=3.0",
     "pandas~=2.0",
+    "floris>=4",
```

Comment:
"~=4.0" would be consistent with other lines, but up to you, FYI ~=4.0 is like >=4, except it won't automatically step up to FLORIS 5, which I think is probably correct

Reply:
Yeah, maybe this is my misunderstanding---should the `.0` part be on there? Does `floris~=4` allow version 3? I'm happy to go with your suggestion on this, I'm not super knowledgeable about semantic versioning

Reply:
RIght sorry!  Yes the .0 is needed, floris~=4.0 in my mind means any version of = or greater to the later decimal of the version you specify, so by saying 4.0, 4.1 is ok but not 5.0, if you say 4.1, then 4.0 is also not ok